packages/twenty-server/src/core/open-api/utils/base-schema.utils.ts

```diff
@@ -0,0 +1,57 @@
+export const baseSchema = (frontBaseUrl: string): { [key: string]: any } => {
```

Comment:
@martmull Is there any library to better type this ? instead of putting any ?

Reply:
yes, need to find this

Reply:
done with [openapi-types](https://www.npmjs.com/package/openapi-types)